src/com/example/homework/homeworkFour/Workers.java

```diff
@@ -0,0 +1,23 @@
+package com.example.homework.homeworkFour;
+
+public class Workers {
+    Employee employee1;
+    Employee employee2;
+
+    public void setParams() {
+        this.employee1 = new Employee(3000, "Green", 1);
```

Comment:
Please create objects and set special values for instance variables in the main method